FordLabs/PeopleMover
# Fixed dashboard welcome message flashing

## Issue
Connects #90 Dashboard Welcome Message

## What was done
- [x] Ensured dashboard welcome message doesn't flash on screen when user has spaces to load

## How to test
1. Go to user dashboard with spaces already created
2. Ensure you do not see the welcome message flash before seeing your spaces
